src/components/DatePicker/DatePicker.jsx

```diff
@@ -84,7 +91,7 @@ export const DatePicker = ({
 
     const isDisabled = (date) => {
         if (isArray(disabledDays)) {
-            return disabledDays.some((_date) => dayjs(_date).isSame(date, "day"));
+            return disabledDays.some((_date) => now(_date).isSame(date, "day"));
```

Comment:
Use a helper method like I mentioned above:
```suggestion
            return disabledDays.some((disabledDate) => isSame(disabledDate, date, "day"));
```
or even:
```suggestion
            return disabledDays.some((disabledDate) => isSameDay(disabledDate, date));
```

Ensure that both dates are of the correct type.